SimPPS/PPSPixelDigiProducer/plugins/PPSPixelDigiProducer.cc

```diff
@@ -0,0 +1,260 @@
+#ifndef SimPPS_PPSPixelDigiProducer_PPSPixelDigiProducer_h
+#define SimPPS_PPSPixelDigiProducer_PPSPixelDigiProducer_h
+
+// -*- C++ -*-
+//
+// Package:    PPSPixelDigiProducer
+// Class:      CTPPSPixelDigiProducer
+//
+/**\class CTPPSPixelDigiProducer PPSPixelDigiProducer.cc SimPPS/PPSPixelDigiProducer/plugins/PPSPixelDigiProducer.cc
+
+ Description: <one line class summary>
+
+ Implementation:
+     <Notes on implementation>
+*/
+//
+// Original Author:  F.Ferro
+//
+
+#include "boost/shared_ptr.hpp"
```

Comment:
@mundim , we use std::shared_ptr everywhere, boost should not be used for that.

Reply:
The code seems to use methods that is not present in the std::shared_ptr, basically the "find" apart from that, moving to std version produces a compilation error concerning a wrong template argument (?). So, if it is mandatory, a longer time will be needed to track down the problems.

Reply:
done